util/Databases/MySQL/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mysql:8.0
```

Comment:
@vgrassia @djsmith85 @addisonbeck.
Also @kspearrin @cscharf :
As also stated in #1470, I think we should rather use a MariaDB container rather than a MySQL one.
MariaDB has widely replaced MySQL in various distributions today.
Thx 👍

Reply:
Please use MariaDB if possible, in most cases it performs identical or better and has less potential future licensing issues.

Reply:
@vgrassia @djsmith85 @addisonbeck.
Also @kspearrin @cscharf :
Did you take this recommandation into consideration ? :)
Many thanks 👍